.gitlab/JOBOWNERS

```diff
@@ -28,6 +28,30 @@ build_serverless*                    @DataDog/serverless
 # Package deps build
 generate_minimized_btfs_*            @DataDog/ebpf-platform
 
+# Package build
+agent_deb*                           @DataDog/agent-build-and-releases
+iot_agent_deb*                       @DataDog/agent-build-and-releases
+dogstatsd_deb*                       @DataDog/agent-build-and-releases
+agent_heroku_deb*                    @DataDog/agent-build-and-releases
+agentless_scanner_deb*               @DataDog/agent-build-and-releases
+updater_deb*                         @DataDog/agent-build-and-releases
+agent_rpm*                           @DataDog/agent-build-and-releases
+iot_agent_rpm*                       @DataDog/agent-build-and-releases
+dogstatsd_rpm*                       @DataDog/agent-build-and-releases
+agentless_scanner_rpm*               @DataDog/agent-build-and-releases
+agent_suse*                          @DataDog/agent-build-and-releases
+iot_agent_suse*                      @DataDog/agent-build-and-releases
+dogstatsd_suse*                      @DataDog/agent-build-and-releases
+agentless_scanner_suse*              @DataDog/agent-build-and-releases
+agent_dmg*                           @DataDog/agent-build-and-releases
+windows_msi*                         @DataDog/windows-agent
+
+# Testing package deploy
+deploy_deb_testing*                  @DataDog/agent-build-and-releases
+deploy_rpm_testing*                  @DataDog/agent-build-and-releases
+deploy_suse_rpm_testing*             @DataDog/agent-build-and-releases
+deploy_windows_testing*              @DataDog/agent-build-and-releases
```

Comment:
Just to double check, doesn't this belong to `windows-agent`?

Reply:
This job doesn't really do anything Windows-specific, it only pushes Windows artifacts to the testing repos. I see it as the testing equivalent of the release tooling for our prod repos.